CHANGELOG.md

```diff
@@ -38,6 +38,14 @@ Bottom level categories:
 - Hal
 -->
 
+By @bradwerth in [#4203](https://github.com/gfx-rs/wgpu/pull/4023)
```

Comment:
This line should be at the end of the section, not before it, and the whole section should go in the Bug Fixes section under `## Unreleased`, not above it.

Reply:
Mid-air collision; I opened a similar conversation about this: https://github.com/gfx-rs/wgpu/pull/4023#discussion_r1289303065